po/pt.po

```diff
@@ -1,14 +1,14 @@
 # Portuguese translation for pantheon-tweaks-plug
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the pantheon-tweaks-plug package.
-# Hugo Carvalho <hugokarvalho@hotmail.com>, 2021, 2022.
+# Hugo Carvalho <hugokarvalho@hotmail.com>, 2021, 2022, 2023.
```

Comment:
The number of years in this line would be `X - 2021 + 1` in X (X is year). This would be even better:

```suggestion
# Hugo Carvalho <hugokarvalho@hotmail.com>, 2021-2023.
```